cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaMirrorMaker2ConnectorsTest.java

```diff
@@ -147,7 +147,66 @@ public void testMirrorTargetClusterNotSameAsConnectCluster() {
                 .build();
         ex = assertThrows(InvalidResourceException.class, () -> KafkaMirrorMaker2Connectors.validateConnectors(kmm2CorrectAndIncorrectMirror));
         assertThat(ex.getMessage(), is("KafkaMirrorMaker2 resource validation failed: " +
-                "[Connect cluster alias (currently set to target) has to be the same as the target cluster alias third]"));
+                "[Connect cluster alias (currently set to target) must match the target cluster alias third or both clusters must have the same bootstrap servers.]"));
+    }
+
+    @Test
+    public void testClusterNotSameButBootstrapUrlSame() {
+        KafkaMirrorMaker2 kmm2 = new KafkaMirrorMaker2Builder(KMM2)
+                .editSpec()
+                        .withConnectCluster("source")
+                .addToClusters(new KafkaMirrorMaker2ClusterSpecBuilder()
+                        .withAlias("third")
+                        .withBootstrapServers("source:9092")
+                        .build())
+                .editMirror(0)
+                        .withTargetCluster("third")
+                .endMirror()
+                .endSpec()
```

Comment:
This should be aligned according to the `edit` / `end` methods. Same applies to other two tests below.

```suggestion
                .editSpec()
                    .withConnectCluster("source")
                        .addToClusters(new KafkaMirrorMaker2ClusterSpecBuilder()
                            .withAlias("third")
                            .withBootstrapServers("source:9092")
                            .build())
                    .editMirror(0)
                        .withTargetCluster("third")
                    .endMirror()
                .endSpec()
```